File: BackEnd/GardenModels/Feedback.cs
namespace Garden.Models
{
    // for database
    public class Feedback
    {
        public string FeedbackId { get; set; }
        public string UserId { get; set; }
        public string? Description { get; set; }
        public DateTime FeedbackTime { get; set; }
    }

    // for data transfer

    public class FeedbackModel
    {
        public string AccountId { get; set; }
        public string SelectedRadio { get; set; }
        public string Topic { get; set; }
        public string Content { get; set; }
        public string Email { get; set; }
    }
}

File: FrontEnd/src/api/Feedback.js
import axios from 'axios';

const API = axios.create({
    baseURL: 'http://39.106.47.60:5194',
    timeout: 3000,
    async: true,
    crossDomain: true,
})

export function postFeedback(user_id, selectedRadio, topic, content, files, email) {
    let feedback = {
        accountId: user_id,
        selectedRadio: selectedRadio, 
        topic: topic,
        content: content,
        email: email
    }

    const formData = new FormData(); 
    formData.append('feedbackJson', JSON.stringify(feedback));
  
    const response = API.post('/feedback/post', formData);
    return response.data;
}